usr/lib/ica_s390_stdll/ica_specific.c

```diff
@@ -3125,6 +3125,31 @@ static CK_RV os_specific_rsa_decrypt(STDLL_TokData_t *tokdata,
 
 }
 
+static CK_BBOOL save_data_in_context(CK_BYTE *in_data, CK_ULONG in_data_len,
+                                     CK_ULONG tag_data_len,
+                                     AES_GCM_CONTEXT *context, CK_BYTE encrypt,
+                                     CK_ULONG *total, CK_ULONG *remain)
```

Comment:
why is `total` a by-reference variable? It is not updated in this routine, so it can be passed by-value.

Reply:
Right, this was a leftover from copying the inline code into the subroutine. 
Done.